rebasehelper/options.py

```diff
@@ -116,6 +117,13 @@
         "default": None,
         "help": "tool to use for determining latest upstream version",
     },
+    {
+        "name": ["--disable-spec-hook"],
```

Comment:
I think better name for this would be `--spec-hook-blacklist`. It would make it obvious that it is a list and one can specify multiple items. What do you think?

Reply:
Yes, blacklist describes it better than disable. I will also use "--versioneer-blacklist" for disabling versionners.